redis/files/redis-2.4.conf.jinja

```diff
@@ -106,423 +41,86 @@ save 300 10
 save 60 10000
```

Comment:
Just asking because you *do* generate other `save 23 42` entries already. It's just that `save 60 10000` would always be present now.